style.css

```diff
@@ -696,53 +707,83 @@ body {
 
 @media (max-width: 768px) {
   #contact {
+    background-color: #fff;
     width: 375px;
   }
 
-  .contact-container {
-    background-image: url(image/contact-form-shapes.png);
-  }
-
-  .contact-me div {
-    display: flex;
-    justify-content: center;
+  #contact .contact-case {
+    background-color: #6070ff;
+    padding: 0 0;
   }
 
-  .contact-container .contact-me {
+  .contact-image {
     display: flex;
     justify-content: center;
+    background-image: url(image/contact-form-shapes.png);
+    background-color: #6070ff;
+    background-repeat: no-repeat;
+    background-size: cover;
+    padding: 150px 0;
+    border-radius: 100px 0 0 0;
   }
 
-  .contact-container .para {
+  .contact-container h2 {
     display: flex;
     justify-content: center;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #fff;
+    margin-bottom: 20px;
   }
 
-  .contact-container .para p {
+  .contact-container p {
     width: 327px;
     text-align: center;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #ebebff;
+    margin-bottom: 20px;
   }
 
-  .contact-container form div li {
+  .contact-container form {
```

Comment:
- [ ] Please as the last reviewer said some of the design parts disappear in different sizes the reason for this is the fixed width you added in the card body you can use CSS to add width for different sizes consider using min-width and max-width property also you can use vw sizes as I said before
![image](https://user-images.githubusercontent.com/74744787/215215917-5e8667a6-e79e-4828-87ba-02114681e786.png)